static/app/views/releases/list/index.tsx

```diff
@@ -244,6 +247,53 @@ class ReleasesList extends AsyncView<Props, State> {
     );
   }
 
+  renderAlertBanner() {
+    const {selection, organization} = this.props;
+
+    const selectedProjectId =
+      selection.projects && selection.projects.length === 1 && selection.projects[0];
+    const selectedProject = organization.projects.find(
+      p => p.id === `${selectedProjectId}`
+    );
+
+    if (!selectedProject) {
+      return null;
+    }
+
+    return (
+      <Feature features={['organizations:release-adoption-chart']}>
+        <Projects orgId={organization.slug} slugs={[selectedProject.slug]}>
+          {({projects, initiallyLoaded, fetchError}) => {
+            const project = projects && projects.length === 1 && projects[0];
+            const projectHasReleases =
+              project &&
+              project.hasOwnProperty('features') &&
+              (project as Project).features.includes('releases');
```

Comment:
I think that we can safely skip this check.

Reply:
👍